ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -87,35 +106,48 @@ def extract_mid_slice_and_convert_coordinates_to_heatmaps(path, suffix, aim=-1):
+            nib_pred = rotate_nifti(nib_pred)
+            nib.save(nib_pred, os.path.join(path, 'derivatives', 'labels', t[i], 'anat', t[i]+'_rec-mid' +suffix+ '_heatmap'+str(aim)
+                                            + '.nii.gz'))
         else:
             pass
 
 
 def get_parser():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-p", "--path", dest="path", required=True, type=str,
-                        help="Path to bids folder",
-                        metavar=imed_utils.Metavar.file)
-    parser.add_argument("-s", "--suffix", dest="suffix", required=True, type=str,
-                        help="""Suffix of the input file as in
-                                sub-xxxSUFFIX.nii.gz (E.g., _T2w)""",
-                        metavar=imed_utils.Metavar.str)
-    parser.add_argument("-a", "--aim", dest="aim", default=-1, type=int,
-                        help="""-1 or positive int. If set to any positive int,
-                                only label with this value will be taken into account""",
-                        metavar=imed_utils.Metavar.int)
+    parser = argparse.ArgumentParser(
+        formatter_class = argparse.RawDescriptionHelpFormatter,
+        description = textwrap.dedent("""
+            This function takes as input a path to a dataset and generates a set of images:
+                (i) mid-sagittal image and
+                (ii) heatmap of disc labels associated with the mid-sagittal image. Please refer to for more details. 
+            
+            Example:
+                python3 prepare_dataset_vertebral_labeling.py --path /data/data-multi-subject/ -s _T2w
+            
+            Reference:
+                https://github.com/neuropoly/vertebral-labeling-deep-learning
+                """)
+    )
+    parser.add_argument("-p", "--path", dest="path", required=False, type=str,
+                        help="Path to bids folder")
+    parser.add_argument("-s", "--suffix", dest="suffix", required=False,
+                        type=str, help="Suffix of the input file as in sub-xxxSUFFIX.nii.gz (E.g., _T2w)")
+    parser.add_argument("-a", "--aim", dest="aim", default=0, type=int,
+                        help="-1 or positive int. If set to any positive int,"
+                             " only label with this value will be taken into account ")
     return parser
 
 
-def main(args=None):
+def main():
     imed_utils.init_ivadomed()
-    parser = get_parser()
-    args = imed_utils.get_arguments(parser, args)
-    extract_mid_slice_and_convert_coordinates_to_heatmaps(path=args.path, suffix=args.suffix,
-                                                          aim=args.aim)
 
-
-if __name__ == '__main__':
+    parser = get_parser()
+    args = parser.parse_args()
+    bids_path = args.path
+    suffix = args.suffix
+    aim = args.aim
+    # Run Script
+    extract_mid_slice_and_convert_coordinates_to_heatmaps(bids_path, suffix, aim)
+
+if __name__=='__main__':
     main()
```

Comment:
Please revert these changes. This is removing functionality from the script, and is unrelated to the main focus of your PR.

Reply:
@rezazad68 This was marked as resolved, but could you please double-check the file? I still see some changes that I don't think should be there:

![Screenshot from 2021-07-05 10-58-57](https://user-images.githubusercontent.com/16181459/124490398-0ab33080-dd80-11eb-94d4-1925dd2d5738.png)

Reply:
If I could make a suggestion, I think it would be good to try looking at the "[Files changed](https://github.com/ivadomed/ivadomed/pull/787/files)" view after you push changes, just to double-check to make sure everything looks good before marking as resolved.

Reply:
Done.